swatch-billable-usage/src/main/resources/META-INF/openapi.yaml

```diff
@@ -197,6 +197,23 @@ paths:
           $ref: "../../../../../spec/error-responses.yaml#/$defs/Unauthorized"
         '500':
           $ref: "../../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+
+  /api/swatch-billable-usage/internal/rpc/remittance/reconcile:
+    post:
+      operationId: reconcileRemittances
+      summary: 'Reconcile existing remittances matching the configured stuck policy.'
+      responses:
```

Comment:
With the securitySchemes in this file, a 403 response type could be added.  It would be consistent with several other defined endpoints.